lib/index.js

```diff
@@ -242,6 +242,19 @@ const resourcesProperties = {
   parallelTitle: mappingTemplates.fulltextFolded,
   parallelTitleDisplay: mappingTemplates.fulltextFolded,
   parallelSeriesStatement: mappingTemplates.fulltextWithRawFolded,
+  parallelTitleAlt: mappingTemplates.fulltextFolded,
+  parallelSubjectLiteral: mappingTemplates.fulltextFolded,
+  parallelUniformTitle: mappingTemplates.fulltextFolded,
+  parallelCreatorLiteral: mappingTemplates.fulltextFolded,
+  parallelContributorLiteral: mappingTemplates.fulltextFolded,
```

Comment:
For literals that we might want to [run filters on](https://github.com/NYPL/discovery-api/blob/3d97adc8035a016acd3e8b1cf2af53a285904f4b/lib/resources.js#L117-L118) in the future, we should include a "raw" field, so let's use `mappingTemplates.fulltextWithRawFolded` for those fields whose primary field is indexed in that way:
```
  parallelCreatorLiteral: mappingTemplates.fulltextWithRawFolded,
  parallelContributorLiteral: mappingTemplates.fulltextWithRawFolded,
```

This will enable us to activate the parallel creator and contributor values we display as links to the Search page with those filters

Reply:
Not all fields. Just the creator and contributor.

Reply:
Ah sorry, fixed that

Reply:
Looks like there's still a `parallelTitleAlt: mappingTemplates.fulltextWithRawFolded`

Reply:
Thanks for catching that. Hopefully correct now